Updated GeoMessage Adapter for GeoEvent 11.1

yujingwu1997 opened the pull request:
Updated properties in pom file and added additional ones for Gallery item discovery in 11.1
Added item.json file for Gallery item discovery in 11.1 
Updated description in properties file to include release number

yujingwu1997 committed:
Updated properties in pom file and added additional ones for Gallery item discovery in 11.1
Added item.json file for Gallery item discovery in 11.1
Updated description in properties file to include release number

yujingwu1997 committed:
Include geoevent definitions for different geomessage types

yujingwu1997 committed:
Replace item.json and update property names in pom.xml

1. Update item.json with releaseItem.json and component.json
2. Update property names in pom.xml file to align with those in component.json and releaseItem.json

yujingwu1997 committed:
1. Update component.json and releaseItem.json to add additional parameters
2. Update pom file to remove unused encoding

eironside reviewed (approved):
Please review updated Gallery Item design docs one more time to make sure this project is ok before merging

yujingwu1997 committed:
Update for 11.1 Gallery Discovery

1. Update releaseItem.json and component.json to latest version
2. Update pom.xml to add additional properties needed for gallery discovery

yujingwu1997 committed:
Update labels and property name of GeoMessage Adapter
